syntaxes/vscode-scl.tmLanguage.json

```diff
@@ -143,7 +143,7 @@
 					"name": "support.function.scl"
 				},
 				{
-					"match": "(?i)\\b(S7_)\\w+",
+					"match": "(?i)\\b(S7_)\\w+|ExternalAccessible|ExternalVisible|ExternalWritable|LibVersion|InstructionName",
```

Comment:
Is there a reason you want to be specific here? Here [/snippets/vscode-scl.json#L570 ](https://github.com/Gunders89/vscode-scl/blob/64ead9a3a494258228fa16aedfdceb8764fcb842/snippets/vscode-scl.json#L570 ) I've added information to each one , but was unsure if it was necassary to do this.

Reply:
Correction, those you mention here ExternalAccessible|ExternalVisible|ExternalWritable|LibVersion|InstructionName are not covered in /snippets/vscode-scl.json

Reply:
Just to clarify, I want them with to highlight the declaration section where they are used: 
![image](https://github.com/user-attachments/assets/fe425fa9-7976-46ad-bd60-7e4c2991856b)

Reply:
I see, but to make it more readable this could be it's own element.

```
.
.
.},
{
    "match": "(?i)\\b(ExternalAccessible|ExternalVisible|ExternalWritable|LibVersion|InstructionName)",
    "name": "support.variable.scl"
},
.
.
.
```

Reply:
Fair enough, updated it now.